spec/models/user_spec.rb

```diff
@@ -1,8 +1,7 @@
 require 'rails_helper'
 
 RSpec.describe User, type: :model do
-  let(:user) { FactoryBot.build(:user) }
-  let(:project) { FactoryBot.build(:project, user: user) }
+  subject(:user) { FactoryBot.build(:user) }
 
   it 'factory is valid' do
     user = FactoryBot.build(:user)
```

Comment:
We repeat the building of a factory a lot throughout this spec. Now that I've introduced a `subject` we could remove the building of factories. For example, this first test could now refactored to:
```
it { is_expected.to be_valid }
```
And I could do the same for the rest of the tests. What do you reckon?